feature-libs/storefinder/components/store-finder-store-description/store-finder-store-description.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container *ngIf="location">
   <div class="container">
     <div class="row">
-      <article class="cx-store col-md-4">
+      <article class="cx-store col-lg-4">
```

Comment:
I also see some opportunity for clean up here and on line `63`. We would like to remove css bootstrap "one day". But that will be breaking change armageddon at this point. 

We can however start working towards that goal by leveraging those `cx-store` and `cx-storeMap` classes to have the breakpoints in scss instead of relying on those `col-lg-X` bootstrap classes.

Reply:
I got rid of bootstrap here altogether and did some clean up. If this is too out of scope let me know!